### PR TITLE
Created a checkout component

### PR DESCRIPTION
**Pull request by ReinisZogots**

### CODE REVIEW

File: src/app/components/checkout/checkout.component.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <h5 class="checkout__backButton__title">Back</h5>
+  </div>
+  <div class="checkout__item__container">
+    <div class="checkout__item" *ngFor="let orderItem of order.items">
+      <img
+        class="checkout__item__cover"
+        src="{{ orderItem.book.path }}"
+        alt="book cover picture"
+      />
+      <div class="checkout__item__title-price-container">
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.html:13]: I would suggest renaming it, seems quite long name. Maybe something like sub container

File: src/app/components/checkout/checkout.component.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      </div>
+    </div>
+  </div>
+  <div class="checkout__totalCost__container">
+    <div class="checkout__header">Total cost: </div>
+    <div class="checkout__totalCost">{{ order.totalPrice | number:'1.2-2'}}€</div>
+  </div>
+  <form [formGroup]="checkoutForm" class="checkout__checkoutForm" (ngSubmit)="createOrder()">
+    <div class="checkout__checkoutForm__inputField">
+      <label class="checkout__checkoutForm__inputField__label">email:</label>
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.html:25]: These class names for checkout form seem long too, maybe can shorten it?

File: src/app/components/checkout/checkout.component.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <h6 class="checkout__item__title">{{ orderItem.book.title }}</h6>
+        <span class="checkout__item__price">{{ orderItem.price | number:'1.2-2'}}€</span>
+      </div>
+    </div>
+  </div>
+  <div class="checkout__totalCost__container">
+    <div class="checkout__header">Total cost: </div>
+    <div class="checkout__totalCost">{{ order.totalPrice | number:'1.2-2'}}€</div>
+  </div>
+  <form [formGroup]="checkoutForm" class="checkout__checkoutForm" (ngSubmit)="createOrder()">
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.html:23]: Can shorten at least checkoutForm to just form

File: src/app/components/checkout/checkout.component.scss [hunk 4]
```diff
@@ -0,0 +1,104 @@
+.checkout {
+  font-family: "Montserrat";
+  &__backButton {
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.scss:3]: Add empty line

File: src/app/components/checkout/checkout.component.scss [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+
+    &__title-price-container {
+      width: 100%;
+      align-items: center;
+      justify-content: space-between;
+      display: flex;
+    }
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.scss:35]: Add empty line

File: src/app/components/checkout/checkout.component.scss [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    &__cover {
+      height: 50px;
+      margin: 10px;
+    }
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.scss:39]: Add empty line

File: src/app/components/checkout/checkout.component.scss [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    &__title {
+      font-size: 10px;
+      margin: 10px;
+      max-width: 50ch;
+      overflow: hidden;
+    }
```
Comment (EgijaLocmele) [inline src/app/components/checkout/checkout.component.scss:45]: Add empty line

File: src/app/components/checkout/checkout.component.html (hunk 1 above)
Comment (ReinisZogots) [reply src/app/components/checkout/checkout.component.html:13]: renamed to checkout__item__subcontainer

File: src/app/components/checkout/checkout.component.html (hunk 2 above)
Comment (ReinisZogots) [reply src/app/components/checkout/checkout.component.html:25]: renamed to  checkout__form__field__label

File: src/app/cart.service.ts [hunk 8]
```diff
@@ -41,4 +41,9 @@ export class CartService {
   getCart(): Cart {
     return this.cart;
   }
+
+  clearCart(): void {
```
Comment (Jalal999) [inline src/app/cart.service.ts:45]: give access modifiers to your functions

File: src/app/components/checkout/checkout.component.ts [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+@Component({
+  selector: 'app-checkout',
+  templateUrl: './checkout.component.html',
+  styleUrls: ['./checkout.component.scss'],
+})
+export class CheckoutComponent implements OnInit {
+  order: Order = new Order();
+  checkoutForm: FormGroup;
+  orderIsPlaced = false;
```
Comment (Jalal999) [inline src/app/components/checkout/checkout.component.ts:16]: access modifier

**Review by EgijaLocmele — APPROVED** [review]

**Pull request merged**